src/domains/ioweb-common/06_cdn_itn.tf

```diff
@@ -47,3 +52,159 @@ module "io_web_profile_itn_fe_st" {
 
   tags = var.tags
 }
+
+resource "azurerm_cdn_frontdoor_profile" "portal_profile" {
+  name                = format("%s-profile-portal-afd-01", local.project_itn)
+  resource_group_name = azurerm_resource_group.io_web_profile_itn_fe_rg.name
+  sku_name            = "Standard_AzureFrontDoor"
+
+  tags = var.tags
+}
+
+data "azurerm_key_vault_certificate" "portal_custom_certificate" {
+  name         = "account-ioapp-it"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_cdn_frontdoor_secret" "portal_certificate" {
+  name                     = "certificate"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  secret {
+    customer_certificate {
+      key_vault_certificate_id = data.azurerm_key_vault_certificate.portal_custom_certificate.id
```

Comment:
```suggestion
      key_vault_certificate_id = data.azurerm_key_vault_certificate.portal_custom_certificate. versionless_id
```

Reply:
I changed the certificate type with a Managed one in https://github.com/pagopa/io-infra/pull/1341/commits/4cad3e946adfbaa1ee253b1c0bf10446529de13d